addons/udes_stock/models/stock_warehouse.py

```diff
@@ -84,6 +84,11 @@ def _domain_warehouse_picking_types(self):
         string="Stock Investigation Picking Type",
         help="Picking type used to create stock investigation pickings.",
     )
+    u_goods_in_picking_type_id = fields.Many2one(
+        comodel_name="stock.picking.type",
+        string="Goods In Picking Type",
+        help="Goods in picking type used for finding the correct picking type from a the warehouse.",
```

Comment:
Nit: `from a the`.

TBH I think just `Goods In picking type for this warehouse` is sufficient.

Reply:
This should be good now thanks.